local.Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang:alpine as BUILD
```

Comment:
Can this be `golang:1.18-alpine` to match CI?

Reply:
go mod is still 1.17, but happy to make everything 1.18 in another PR

Reply:
We are building with 1.18 in CI but deliberately using `go 1.17` in go.mod.

The `go 1.17` in go.mod means that we are not allowing the project to use new language features introduced in go1.18 or later.

This ensures that external contributors who may have to rely on the stable version of Go can be reasonably sure that they can continue to build relayer. 

That being said, I am not opposed to keeping go.mod and CI using the current, rather than the stable, version of Go -- because relayer is an application we deliver, not a library that is intended to be imported into other projects.